src/blocks/stars.scss

```
@@ -0,0 +1,20 @@
+.stars {
+  display: flex;
+
+  &__star {
+    background-position: center;
+    background-image: url(../images/star-active.svg);
+    background-repeat: no-repeat;
+    height: 16px;
+    width: 16px;
+    margin-right: 4px;
+  
+    &:nth-child(n+5) {
+      background-image: url(../images/star.svg);
+    }
+  
+    &:last-child {
+      margin-right: 15px;
+    }
+  }
```

I suppose you want to make the last star inactive. but what if you have a class for 3 active stars instead of 4: `stars stars--4` ?
it's better to set the default star image for `star` class and add dynamic CSS styles for active
```suggestion

  &__star {
    background-position: center;
    background-image: url(../images/star.svg);
    background-repeat: no-repeat;
    height: 16px;
    width: 16px;
    margin-right: 4px;
  
    &:last-child {
      margin-right: 15px;
    }
  }
  
  @for $starCount from 1 through 5 {
    &--#{$starCount} .stars__star:nth-child(-n + #{$starCount}) {
      background-image: url(../images/star-active.svg);
    }
  }
```

---

cool thank you